maintainers/maintainer-list.nix

```diff
@@ -4603,6 +4603,16 @@
     githubId = 26020062;
     name = "lumi";
   };
+  lunik1 = {
```

Comment:
The creation of this entry should be a separate commit

Reply:
Roger Roger